### PR TITLE
Announcement Card & Event Card

### LINKED ISSUE #71: [Home Screen] Create Announcement card component

### PR DESCRIPTION
**Pull request by Mtessier809**
[Home Screen] Create Event Card 
[Home Screen] Create Announcement Card
 
Mathis Tessier 

Fixes #71 #72 
Creates event card and announcement card 
## Expected Behavior
Created Event Card inside of components that consists of a card header and card body. Depending on the data passed to the event card, it is generated differently. So far it can be used to generate event posts as well as announcements. 

Example Event Card 

eventCard

### CODE REVIEW

File: frontend/VolunteerOne/navigation/Screens.js [hunk 1]
```diff
@@ -158,12 +158,12 @@ function HomeStack(props) {
       }}
     >
       <Stack.Screen
-        name="Home"
+        name="Announcments"
```
Comment (cerasamson) [inline frontend/VolunteerOne/navigation/Screens.js:161]: Fix typo :-)

File: frontend/VolunteerOne/screens/Home/index.js [hunk 2]
```diff
@@ -1,64 +1,58 @@
 import React from "react";
 import { StyleSheet, Dimensions, ScrollView } from "react-native";
 import { Block, theme } from "galio-framework";
-
-import { Button, Card } from "../../components";
-import articles from "../../constants/articles";
+import EventCard from "../../components/EventCard";
 const { width } = Dimensions.get("screen");
 
+const dataList = [
```
Comment (cerasamson) [inline frontend/VolunteerOne/screens/Home/index.js:7]: I'm not sure if we want to have the dataList in the home page itself or in ```/constants``` (as ```events.js```) for readability purposes, but we can discuss

File: frontend/VolunteerOne/screens/Home/index.js [hunk 3]
```diff
@@ -1,64 +1,58 @@
...
+    location: "",
+    timePosted: new Date("2023-02-02T00:00:00"),
+    type: "event",
+  },
+  {
+    id: 2,
+    organization: "Global Volunteers",
+    announcement:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et enim ad minim veniam, quis nostrud exercitation ullamco laboris ",
+    type: "announcment",
```
Comment (cerasamson) [inline frontend/VolunteerOne/screens/Home/index.js:37]:
```suggestion
    type: "announcement",
```

File: frontend/VolunteerOne/components/EventCard/CardBody.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+          {data["description"]}
+          {"\n\n"}
+          When: {data["date"]} {"\n\n"}
+          Where: {data["location"]} {"\n"}
+        </Text>
+        <Text style={styles.eventLink}>Click to view event</Text>
+      </Block>
+    );
+    //body for an announcment posting//
+  } else if (type == "announcment") {
```
Comment (cerasamson) [inline frontend/VolunteerOne/components/EventCard/CardBody.js:35]:
```suggestion
  } else if (type == "announcement") {
```

**Comment by cerasamson** [comment]
As we discussed in the meeting earlier, we can probably combine styling from #88 and #89 to ```CardBody.js``` and ```CardHeader.js```.

**Review by carmen-lee — CHANGES_REQUESTED** [review]
Thanks for doing the dynamic dates! Looks great 👍 

- Fix typos (announcment -> announcement)
- create file in the folder 'constants' and put the mock data there

**Comment by cerasamson** [comment]
Will be combined with another PR - #97

**Pull request closed without merging**